### PR TITLE
fix(DB): spawn missing vanilla rares

### PR DESCRIPTION
**Pull request by T1ti**
## Changes Proposed:
Checked every single rare creature from vanilla in the database(ID under 18k and rank = 4), checked each entry that doesn't have an entry in the creature table(spawned in game). Checked those individualy and looked up in wowhead if they should be spawned or not, found a few that are unused by blizzard and shouldn't be spawned, all the ones I'm adding are verified. There are 10 missing rares total

-  Spawned 10 missing rare creatures in vanilla zones, mostly in Arathi Highlands and Dustwallow Marsh. Added multiple spawn points(pooling) and waypoints for some, credits to vmangos.
- Also verified thir spells and added a few missing spells in some cases : 
-added cleave and berserker stance to Nimar the slayer 
-add two missing spells : Wither touch and shadowbolt volley to Zalas witherbark and correct his timers based on vmangos
-Added missing spell vicious bite to Grimmaw.

## Tests Performed:
- Verified that the rares are correctly spawned

## How to Test the Changes:
`.go creature xxx`
Here's a list of GUIDs to test, if some creatures have multiple, they use pooling, only one GUID should sapwned at a time for a creature.

1. Kovork : 300753

1. molok the crusher :  300754

1. zalas witherbark : 
300757
301300
301301
301302

1. nimar the slayer : 
300755
301290
301291
301292

1. drogoth the roamer : 
300769
300770

1. ripscale : 
300775
300776

1. The rot : 300773

2. Lord angler : 300771

1. Oozeworm : 300772

1. grimmaw : 300777

## Known Issues and TODO List:
Maybe loots, haven't tested it at all.

## Target Branch(es):
- [x] Master

 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1614130170345582600.sql [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+(@ENTRY, 0, 0, 0, 0, 0, 80, 0, 5000, 5000, 15000, 25000, 11, 512, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, "Every 15 - 25 seconds (5 - 5s initially)  - Self: Cast spell Chains of Ice (512) on Victim"),
+(@ENTRY, 0, 1, 0, 0, 0, 80, 0, 3000, 3000, 15000, 15000, 11, 851, 1, 0, 0, 0, 0, 6, 0, 0, 0, 0, 0, 0, 0, "Every 15 seconds (3s initially)  - Self: Cast spell Polymorph: Sheep (851) on Random hostile (not top) (flags: interrupt previous)"),
+(@ENTRY, 0, 2, 0, 60, 0, 100, 0, 1000, 3000, 5000, 15000, 11, 4974, 32, 0, 0, 0, 0, 5, 0, 0, 0, 0, 0, 0, 0, "Every 5 - 15 seconds (1 - 3s initially) - Self: Cast spell Wither Touch (4974) on Random hostile (flags: aura not present)"),
+(@ENTRY, 0, 3, 0, 60, 0, 100, 0, 5000, 8000, 8000, 15000, 11, 9081, 0, 0, 0, 0, 0, 5, 0, 0, 0, 0, 0, 0, 0, "Every 8 - 15 seconds (5 - 8s initially) - Self: Cast spell Shadow Bolt Volley (9081) on Random hostile");
+
+
+-- nimar the slayer
+-- TODO : he has a script or emote where he eats meat
+INSERT INTO `creature` (`guid`, `id`, `map`, `position_x`, `position_y`, `position_z`, `orientation`, `spawntimesecs`, `wander_distance`, `MovementType`) VALUES
```
Comment (FrancescoBorzi) [inline data/sql/updates/pending_db_world/rev_1614130170345582600.sql:60]: missing `DELETE` before `INSERT`

**Comment by Zoidwaffle** [comment]
I took a peek at this, since I'm interested in this feature myself... But I can't figure out how to make suggestions directly here (I might be retarded or maybe I don't have access).

As an alternative, I can also just post my entire file here so you can import it locally if it makes sense.

File: data/sql/updates/pending_db_world/rev_1614130170345582600.sql [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+(@ENTRY, 0, 0, 0, 0, 0, 80, 0, 5000, 5000, 15000, 25000, 11, 512, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, "Every 15 - 25 seconds (5 - 5s initially)  - Self: Cast spell Chains of Ice (512) on Victim"),
+(@ENTRY, 0, 1, 0, 0, 0, 80, 0, 3000, 3000, 15000, 15000, 11, 851, 1, 0, 0, 0, 0, 6, 0, 0, 0, 0, 0, 0, 0, "Every 15 seconds (3s initially)  - Self: Cast spell Polymorph: Sheep (851) on Random hostile (not top) (flags: interrupt previous)"),
+(@ENTRY, 0, 2, 0, 60, 0, 100, 0, 1000, 3000, 5000, 15000, 11, 4974, 32, 0, 0, 0, 0, 5, 0, 0, 0, 0, 0, 0, 0, "Every 5 - 15 seconds (1 - 3s initially) - Self: Cast spell Wither Touch (4974) on Random hostile (flags: aura not present)"),
+(@ENTRY, 0, 3, 0, 60, 0, 100, 0, 5000, 8000, 8000, 15000, 11, 9081, 0, 0, 0, 0, 0, 5, 0, 0, 0, 0, 0, 0, 0, "Every 8 - 15 seconds (5 - 8s initially) - Self: Cast spell Shadow Bolt Volley (9081) on Random hostile");
+
+
+-- nimar the slayer
+-- TODO : he has a script or emote where he eats meat
```
Comment (Zoidwaffle) [inline data/sql/updates/pending_db_world/rev_1614130170345582600.sql:59]:
```suggestion
-- TODO : he has a script or emote where he eats meat
DELETE FROM `creature` WHERE `guid` IN (300755,301290,301291,301292);
```

File: data/sql/updates/pending_db_world/rev_1614130170345582600.sql [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+INSERT INTO `creature` (`guid`, `id`, `map`, `position_x`, `position_y`, `position_z`, `orientation`, `spawntimesecs`, `wander_distance`, `MovementType`) VALUES
+(300755, 2606, 0, -1902.16, -3193.09, 56.5808, 5.63212, 27000, 0, 0),
+(301290, 2606, 0, -1810.14, -3417.2, 45.4232, 0.985671, 27000, 0, 0),
+(301291, 2606, 0, -1701.21, -3509.16, 60.2556, 6.26748, 27000, 5, 1),
+(301292, 2606, 0, -1672.56, -3246.93, 25.7297, 4.51998, 27000, 0, 2);
+INSERT INTO `pool_creature` (`guid`, `pool_entry`, `chance`, `description`) VALUES
```
Comment (Zoidwaffle) [inline data/sql/updates/pending_db_world/rev_1614130170345582600.sql:65]:
```suggestion
DELETE FROM `pool_creature` WHERE `guid` IN (300755,301290,301291,301292);
INSERT INTO `pool_creature` (`guid`, `pool_entry`, `chance`, `description`) VALUES 
```

File: data/sql/updates/pending_db_world/rev_1614130170345582600.sql [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+(300755, 2606, 0, -1902.16, -3193.09, 56.5808, 5.63212, 27000, 0, 0),
+(301290, 2606, 0, -1810.14, -3417.2, 45.4232, 0.985671, 27000, 0, 0),
+(301291, 2606, 0, -1701.21, -3509.16, 60.2556, 6.26748, 27000, 5, 1),
+(301292, 2606, 0, -1672.56, -3246.93, 25.7297, 4.51998, 27000, 0, 2);
+INSERT INTO `pool_creature` (`guid`, `pool_entry`, `chance`, `description`) VALUES 
+(300755, 2606, 0, 'Nimar the Slayer (Youfie)'),
+(301290, 2606, 0, 'Nimar the Slayer (Youfie)'),
+(301291, 2606, 0, 'Nimar the Slayer (Youfie)'),
+(301292, 2606, 0, 'Nimar the Slayer (Youfie)');
+INSERT INTO `creature_addon` (`guid`, `path_id`, `mount`, `bytes1`, `bytes2`, `emote`, `isLarge`, `auras`) VALUES (301292, 3012920, 0, 0, 1, 0, 0, '');
```
Comment (Zoidwaffle) [inline data/sql/updates/pending_db_world/rev_1614130170345582600.sql:70]:
```suggestion
DELETE FROM `creature_addon` WHERE `guid`=301292;
INSERT INTO `creature_addon` (`guid`, `path_id`, `mount`, `bytes1`, `bytes2`, `emote`, `isLarge`, `auras`) VALUES (301292, 3012920, 0, 0, 1, 0, 0, '');
```

File: data/sql/updates/pending_db_world/rev_1614130170345582600.sql [hunk 5]
```diff
@@ -0,0 +1,140 @@
...
+INSERT INTO `waypoint_data` (`id`, `point`, `position_x`, `position_y`, `position_z`, `orientation`, `delay`) VALUES
```
Comment (Zoidwaffle) [inline data/sql/updates/pending_db_world/rev_1614130170345582600.sql:71]:
```suggestion
DELETE FROM `waypoint_data` WHERE `id`=3012920;
INSERT INTO `waypoint_data` (`id`, `point`, `position_x`, `position_y`, `position_z`, `orientation`, `delay`) VALUES
```

File: data/sql/updates/pending_db_world/rev_1614130170345582600.sql [hunk 6]
```diff
@@ -0,0 +1,140 @@
...
+(@ENTRY, 0, 0, 0, 60, 0, 100, 0, 1000, 1000, 240000, 240000, 11, 2458, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, "Every 240 seconds (1s initially) - Self: Cast spell Berserker Stance (2458) on Self"),
+(@ENTRY, 0, 1, 0, 0, 0, 100, 0, 7800, 10000, 11100, 24200, 11, 17207, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, "Every 11.1 - 24.2 seconds (7.8 - 10s initially)  - Self: Cast spell Whirlwind (17207) on Victim"),
+(@ENTRY, 0, 2, 0, 0, 0, 100, 0, 7000, 13000, 7000, 23000, 11, 15496, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, "Every 7 - 23 seconds (7 - 13s initially)  - Self: Cast spell Cleave (15496) on Victim"),
+(@ENTRY, 0, 3, 0, 12, 0, 100, 1, 0, 30, 0, 0, 11, 7160, 1, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, "When victim health between 0 - 30 (check every 0 - 0) - Self: Cast spell Execute (7160) on Victim (flags: interrupt previous)"),
+(@ENTRY, 0, 4, 0, 2, 0, 100, 1, 0, 15, 0, 0, 25, 1, 0, 0, 0, 0, 0, 7, 0, 0, 0, 0, 0, 0, 0, "When health between 0 and 15 (check every 0 - 0 ms) - Self: Flee for assist");
+
+-- dustwallow marsh
+-- drogoth the roamer
```
Comment (Zoidwaffle) [inline data/sql/updates/pending_db_world/rev_1614130170345582600.sql:98]:
```suggestion
-- drogoth the roamer
DELETE FROM `creature` WHERE `guid` IN (300769,300770);
```

File: data/sql/updates/pending_db_world/rev_1614130170345582600.sql [hunk 7]
```diff
@@ -0,0 +1,140 @@
...
+INSERT INTO `creature` (`guid`, `id`, `map`, `position_x`, `position_y`, `position_z`, `orientation`, `spawntimesecs`, `wander_distance`, `MovementType`) VALUES
+(300769, 14231, 1, -2608.36, -3011.72, 29.4168, 0.0195422, 27000, 10, 1),
+(300770, 14231, 1, -2799.72, -3129.45, 28.7032, 4.41743, 27000, 5, 1);
+INSERT INTO `pool_creature` (`guid`, `pool_entry`, `chance`, `description`) VALUES (300769, 42939, 0, 'Dustwallow Marsh: Drogoth the Roamer');
+INSERT INTO `pool_creature` (`guid`, `pool_entry`, `chance`, `description`) VALUES (300770, 42939, 0, 'Dustwallow Marsh: Drogoth the Roamer');
```
Comment (Zoidwaffle) [inline data/sql/updates/pending_db_world/rev_1614130170345582600.sql:103]:
```suggestion
DELETE FROM `pool_creature` WHERE `guid` IN (300769,300770);
INSERT INTO `pool_creature` (`guid`, `pool_entry`, `chance`, `description`) VALUES
(300769, 42939, 0, 'Dustwallow Marsh: Drogoth the Roamer'),
(300770, 42939, 0, 'Dustwallow Marsh: Drogoth the Roamer');
```

File: data/sql/updates/pending_db_world/rev_1614130170345582600.sql [hunk 8]
```diff
@@ -0,0 +1,140 @@
...
+-- ripscale
```
Comment (Zoidwaffle) [inline data/sql/updates/pending_db_world/rev_1614130170345582600.sql:104]:
```suggestion
-- ripscale
DELETE FROM `creature` WHERE `guid` IN (300775,300776);
```

File: data/sql/updates/pending_db_world/rev_1614130170345582600.sql [hunk 9]
```diff
@@ -0,0 +1,140 @@
...
+INSERT INTO `creature` (`guid`, `id`, `map`, `position_x`, `position_y`, `position_z`, `orientation`, `spawntimesecs`, `wander_distance`, `MovementType`) VALUES
+(300775, 14233, 1, -3969.57, -3182.07, 28.8974, 3.08685, 38000, 10, 1),
+(300776, 14233, 1, -4040.93, -3557.63, 28.5225, 5.65789, 38000, 10, 1);
+INSERT INTO `pool_creature` (`guid`, `pool_entry`, `chance`, `description`) VALUES (300775, 43157, 0, 'Dustwallow Marsh: Ripscale');
+INSERT INTO `pool_creature` (`guid`, `pool_entry`, `chance`, `description`) VALUES (300776, 43157, 0, 'Dustwallow Marsh: Ripscale');
```
Comment (Zoidwaffle) [inline data/sql/updates/pending_db_world/rev_1614130170345582600.sql:109]: DELETE FROM `pool_creature` WHERE `guid` IN (300769,300770);
INSERT INTO `pool_creature` (`guid`, `pool_entry`, `chance`, `description`) VALUES
(300769, 42939, 0, 'Dustwallow Marsh: Drogoth the Roamer'),
(300770, 42939, 0, 'Dustwallow Marsh: Drogoth the Roamer');

File: data/sql/updates/pending_db_world/rev_1614130170345582600.sql [hunk 10]
```diff
@@ -0,0 +1,140 @@
...
+INSERT INTO `creature` (`guid`, `id`, `map`, `position_x`, `position_y`, `position_z`, `orientation`, `spawntimesecs`, `wander_distance`, `MovementType`) VALUES
+(300769, 14231, 1, -2608.36, -3011.72, 29.4168, 0.0195422, 27000, 10, 1),
+(300770, 14231, 1, -2799.72, -3129.45, 28.7032, 4.41743, 27000, 5, 1);
+INSERT INTO `pool_creature` (`guid`, `pool_entry`, `chance`, `description`) VALUES (300769, 42939, 0, 'Dustwallow Marsh: Drogoth the Roamer');
+INSERT INTO `pool_creature` (`guid`, `pool_entry`, `chance`, `description`) VALUES (300770, 42939, 0, 'Dustwallow Marsh: Drogoth the Roamer');
+-- ripscale
+INSERT INTO `creature` (`guid`, `id`, `map`, `position_x`, `position_y`, `position_z`, `orientation`, `spawntimesecs`, `wander_distance`, `MovementType`) VALUES
+(300775, 14233, 1, -3969.57, -3182.07, 28.8974, 3.08685, 38000, 10, 1),
+(300776, 14233, 1, -4040.93, -3557.63, 28.5225, 5.65789, 38000, 10, 1);
+INSERT INTO `pool_creature` (`guid`, `pool_entry`, `chance`, `description`) VALUES (300775, 43157, 0, 'Dustwallow Marsh: Ripscale');
```
Comment (Zoidwaffle) [inline data/sql/updates/pending_db_world/rev_1614130170345582600.sql:108]:
```suggestion
DELETE FROM `pool_creature` WHERE `guid` IN (300769,300770);
INSERT INTO `pool_creature` (`guid`, `pool_entry`, `chance`, `description`) VALUES
(300769, 42939, 0, 'Dustwallow Marsh: Drogoth the Roamer'),
(300770, 42939, 0, 'Dustwallow Marsh: Drogoth the Roamer');
```

File: data/sql/updates/pending_db_world/rev_1614130170345582600.sql (hunk 9 above)
Comment (Zoidwaffle) [inline data/sql/updates/pending_db_world/rev_1614130170345582600.sql:109]:
```suggestion
DELETE FROM `pool_creature` WHERE `guid` IN (300775,300776);
INSERT INTO `pool_creature` (`guid`, `pool_entry`, `chance`, `description`) VALUES
(300775, 43157, 0, 'Dustwallow Marsh: Ripscale'),
(300776, 43157, 0, 'Dustwallow Marsh: Ripscale');
```

File: data/sql/updates/pending_db_world/rev_1614130170345582600.sql [hunk 11]
```diff
@@ -0,0 +1,140 @@
...
+INSERT INTO `pool_creature` (`guid`, `pool_entry`, `chance`, `description`) VALUES (300776, 43157, 0, 'Dustwallow Marsh: Ripscale');
+-- The rot
+INSERT INTO `creature` (`guid`, `id`, `map`, `position_x`, `position_y`, `position_z`, `orientation`, `spawntimesecs`, `wander_distance`, `MovementType`) VALUES (300773, 14235, 1, -4042.09, -3753.6, 43.2492, 6.26622, 108000, 5, 1);
+-- lord angler
+INSERT INTO `creature` (`guid`, `id`, `map`, `position_x`, `position_y`, `position_z`, `orientation`, `spawntimesecs`, `wander_distance`, `MovementType`) VALUES (300771, 14236, 1, -4250.64, -3849.09, -6.71606, 5.12114, 108000, 5, 1);
+-- oozeworm
+INSERT INTO `creature` (`guid`, `id`, `map`, `position_x`, `position_y`, `position_z`, `orientation`, `spawntimesecs`, `wander_distance`, `MovementType`) VALUES (300772, 14237, 1, -4207.99, -2911.04, 44.9497, 2.051, 180000, 5, 1);
```
Comment (Zoidwaffle) [inline data/sql/updates/pending_db_world/rev_1614130170345582600.sql:115]:
```suggestion
-- The rot
DELETE FROM `creature` WHERE `guid`=300773;
INSERT INTO `creature` (`guid`, `id`, `map`, `position_x`, `position_y`, `position_z`, `orientation`, `spawntimesecs`, `wander_distance`, `MovementType`) VALUES (300773, 14235, 1, -4042.09, -3753.6, 43.2492, 6.26622, 108000, 5, 1);
-- lord angler
DELETE FROM `creature` WHERE `guid`=300771;
INSERT INTO `creature` (`guid`, `id`, `map`, `position_x`, `position_y`, `position_z`, `orientation`, `spawntimesecs`, `wander_distance`, `MovementType`) VALUES (300771, 14236, 1, -4250.64, -3849.09, -6.71606, 5.12114, 108000, 5, 1);
-- oozeworm
DELETE FROM `creature` WHERE `guid`=300772;
INSERT INTO `creature` (`guid`, `id`, `map`, `position_x`, `position_y`, `position_z`, `orientation`, `spawntimesecs`, `wander_distance`, `MovementType`) VALUES (300772, 14237, 1, -4207.99, -2911.04, 44.9497, 2.051, 180000, 5, 1);
```

File: data/sql/updates/pending_db_world/rev_1614130170345582600.sql [hunk 12]
```diff
@@ -0,0 +1,140 @@
...
+INSERT INTO `waypoint_data` (`id`, `point`, `position_x`, `position_y`, `position_z`, `orientation`, `delay`) VALUES (3007770, 8, 9108.77, 1809.06, 1325.75, 4.86517, 0);
+INSERT INTO `waypoint_data` (`id`, `point`, `position_x`, `position_y`, `position_z`, `orientation`, `delay`) VALUES (3007770, 9, 9115.99, 1771.94, 1321.74, 5.14595, 0);
+INSERT INTO `waypoint_data` (`id`, `point`, `position_x`, `position_y`, `position_z`, `orientation`, `delay`) VALUES (3007770, 10, 9136.82, 1754.07, 1318.87, 4.75914, 0);
+INSERT INTO `waypoint_data` (`id`, `point`, `position_x`, `position_y`, `position_z`, `orientation`, `delay`) VALUES (3007770, 11, 9141.64, 1726.7, 1319.16, 4.60991, 0);
+INSERT INTO `waypoint_data` (`id`, `point`, `position_x`, `position_y`, `position_z`, `orientation`, `delay`) VALUES (3007770, 12, 9141.35, 1689.69, 1320.03, 4.60991, 0);
+INSERT INTO `waypoint_data` (`id`, `point`, `position_x`, `position_y`, `position_z`, `orientation`, `delay`) VALUES (3007770, 13, 9131.44, 1665.16, 1320.95, 4.71202, 0);
+INSERT INTO `waypoint_data` (`id`, `point`, `position_x`, `position_y`, `position_z`, `orientation`, `delay`) VALUES (3007770, 14, 9132.59, 1646.61, 1322.61, 4.6806, 0);
```
Comment (Zoidwaffle) [inline data/sql/updates/pending_db_world/rev_1614130170345582600.sql:134]:
```suggestion
DELETE FROM `creature` WHERE `guid`=300777;
INSERT INTO `creature` (`guid`, `id`, `map`, `position_x`, `position_y`, `position_z`, `orientation`, `spawntimesecs`, `wander_distance`, `MovementType`) VALUES (300777, 14429, 1, 9132.61, 1651.7, 1322.14, 4.25837, 9000, 0, 2);
DELETE FROM `creature_addon` WHERE `guid`=300777;
INSERT INTO `creature_addon` (`guid`, `path_id`, `mount`, `bytes1`, `bytes2`, `emote`, `isLarge`, `auras`) VALUES (300777, 3007770, 0, 0, 1, 0, 0, '');
DELETE FROM `waypoint_data` WHERE `id`=3007770;
INSERT INTO `waypoint_data` (`id`, `point`, `position_x`, `position_y`, `position_z`, `orientation`, `delay`) VALUES
(3007770, 1, 9132.81, 1664.76, 1320.77, 4.74146, 0),
(3007770, 2, 9141.27, 1690.01, 1320.05, 1.23858, 0),
(3007770, 3, 9141.58, 1727.6, 1319.16, 1.73339, 0),
(3007770, 4, 9137.61, 1753.82, 1319.01, 1.808, 0),
(3007770, 5, 9116, 1771.33, 1321.66, 2.04362, 0),
(3007770, 6, 9109.14, 1808.41, 1325.54, 1.66663, 0),
(3007770, 7, 9111.24, 1823.84, 1328.53, 4.67864, 0),
(3007770, 8, 9108.77, 1809.06, 1325.75, 4.86517, 0),
(3007770, 9, 9115.99, 1771.94, 1321.74, 5.14595, 0),
(3007770, 10, 9136.82, 1754.07, 1318.87, 4.75914, 0),
(3007770, 11, 9141.64, 1726.7, 1319.16, 4.60991, 0),
(3007770, 12, 9141.35, 1689.69, 1320.03, 4.60991, 0),
(3007770, 13, 9131.44, 1665.16, 1320.95, 4.71202, 0),
(3007770, 14, 9132.59, 1646.61, 1322.61, 4.6806, 0);
```

**Comment by Zoidwaffle** [comment]
OK, so I just tested the feature to leave direct feedback to a PR. All the changes may seem like more changes than they actually are... Just added some "delete" statements and shortened some of the repetitive SQL, so I didn't ad much value as such.

**Comment by T1ti** [comment]
I had all the delete statements as one in the first lines, is it bad ?

**Comment by Zoidwaffle** [comment]
nope, that's perfectly fine... Then I just missed them, sorry.

I just saw the comment from FrancescoBorzi and started from there... I didn't pay much attention to things going on earlier.

So please ignore all dumb suggestions not adding any value. I also did this to try out the "suggest changes" option - and just thought I could help out here and get your PR accepted and merged faster :)

**Comment by Zoidwaffle** [comment]
I checked my suggested changes again, and you are right, some of them where handled in the top of your file. Sorry about that, I promise to try and pay better attention next time and deleted the wrongs I found to be handled already.

But for the remaining suggestions above, I don't see them included in the current version - so you can either just accept as-is or add the various references in the top along with the other delete commands... or just close them if you find them wrong.

If a delete should be in the top or further down - I guess it depends on your personal style and preferences. I kinda like to have them together - so 1 line with delete, then the relevant insert just below. To me it just makes sense. But that's just my personal preference, so do what you prefer.

Thanks a lot for all your high quality error reports btw!

**Comment by T1ti** [comment]
Added your suggestion to make it less repetitive

**Comment by Hacki95** [comment]
Is this ready to be tested or still a WIP?

**Comment by T1ti** [comment]
to be tested

**Comment by Sedatyf** [comment]
@FrancescoBorzi sorry for the ping, but as it is an old PR, I thought you could review this in order to be merged. You can see my test just above this comment.

**Review by FrancescoBorzi — APPROVED** [review]

**Comment by FrancescoBorzi** [comment]
@T1ti good job.

@Sedatyf amazing work with testing, thanks again!

**Pull request merged**